.github/workflows/release.yml

```diff
@@ -0,0 +1,28 @@
+name: Release
+on:
+  push:
+    branches: [master]
+    tags: ["*"]
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: olafurpg/setup-scala@v2
+      - uses: olafurpg/setup-gpg@v2
+      - run: git fetch --unshallow
+      - name: Publish
+        run: |
+          sbt ci-release
+          sbt docs/docusaurusPublishGhpages
+        env:
```

Comment:
Does scalameta-parsers also get published here? If so, should we also include that here?

Reply:
To npm? That module has so far been published manually I think. cc/ @gabro it would be nice to automate that publish step but we can do it separately

Reply:
Yup, to NPM. And sounds good! Was just curious.

Reply:
Yeah, I've always been lazy and I've never got around automating that step, mostly because I'm probably the only one using that package (in AST Explorer) :P 

I agree it would be good to automate it